specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/networkWatcher.json

```diff
@@ -2827,6 +2827,18 @@
         "protocolConfiguration": {
           "$ref": "#/definitions/ProtocolConfiguration",
           "description": "Configuration of the protocol."
+        },
+        "preferredIPVersion": {
```

Comment:
You can re-use definition from network.json to avoid creating duplicate enums?

```json
"preferredIPVersion": {
  "$ref": "./network.json#/definitions/IPVersion",
  "description": "Preferred IP version for connectivity check."
}
```